tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -178,12 +178,16 @@ When(
       if (version !== '1') {
         throw new Error('restoring is only supported for the most recent version')
       }
+      acc[to]['openDetailsPanel'] = openDetailsPanel !== 'false'
```

Comment:
other strings will be considered false
```suggestion
      acc[to]['openDetailsPanel'] = openDetailsPanel === 'true'
```